vagrant-service-manager.gemspec

```diff
@@ -16,4 +16,6 @@ Gem::Specification.new do |spec|
 
   spec.files         = `git ls-files -z`.split("\x0")
   spec.require_paths = ['lib']
+
+  spec.add_dependency 'rubyzip'
```

Comment:
Can you add version as `~> 1.2.0` as if there are systems with version `0.9` or less it will break again.
Refer link https://github.com/rubyzip/rubyzip/issues/90#issuecomment-23547717